src/APIErrorNotification.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useContext } from "react";
+import { APIErrorContext } from './APIErrorProvider';
+import {ReactComponent as CloseIcon} from "./images/icons/ic_close.svg";
+import {ReactComponent as ErrorIcon} from "./images/icons/error-icon.svg";
+
+import './APIErrorNotification.scss';
+
+interface IErrorMessage {
+  message: string,
+  status: number,
+  source: string,
+  title: string,
+  detail: string,
+  request_id?: string,
+}
+
+export const APIErrorNotification = () => {
+  const { error, removeError } = useContext(APIErrorContext);
+
+  const handleCloseModal = () => {
+    removeError()
+  };
+
+  return (
+    <div className="notification">
+      {!!error && error.length > 0 && error.map((el: IErrorMessage) => (
+        <div className="notification__body" key={`${el.detail}_${Math.random().toString(36)}`}>
```

Comment:
I look forward to when we can get error message keys from the API to populate this 😄